outport/process/outportDataProvider.go

```diff
@@ -534,6 +534,7 @@ func (odp *outportDataProvider) filterOutDuplicatedMiniBlocks(miniBlocksFromBody
 	return filteredMiniBlocks, nil
 }
 
+// TODO remove after system test
```

Comment:
can be moved to trace instead of removing

Reply:
will do